test/test_completions.jl

```diff
@@ -389,7 +410,7 @@ end
             context=CompletionContext(;
                 triggerKind=CompletionTriggerKind.Invoked))
         items = JETLS.get_completion_items(state, uri, params)
-        @test any(items) do item
+        @test_broken any(items) do item
```

Comment:
```suggestion
        @test_broken any(items) do item # broken since JL currently seems unable to handle multi-arg `@nospecialize`
```
I found this test case is broken because JL fails to expand it and we're hitting this case:
https://github.com/aviatesk/JETLS.jl/blob/d15bc7ca959fae283c989683e7b0c40af6cd1b68/src/completions.jl#L150

Reply:
Might be worth opening an issue there

Reply:
It appears that JL is making custom definitions for some macros in Base to accommodate its macro expansion system without bootstrapping issue.
For now, we decided to adjust those custom definitions (https://github.com/c42f/JuliaLowering.jl/pull/15), but it seems we really need to improve the current situation regarding macro expansion with JL.